glass/grf/_transformations.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from ._core import ArrayT, corr, dcorr, icorr
+
+
+@dataclass
+class Normal:
```

Comment:
I would prefer better docstrings (explicit "Parameters" and "Returns") for these dataclasses and their methods (including the dunder methods).